articles/proofs-arguments-and-zero-knowledge/chapter_2.md

```diff
@@ -74,9 +74,9 @@ Reason: Time complexity of multiplication of a (n x n) matrix and a vector of si
 
    Consider matrices:
 
-   $`A = \begin{bmatrix} 1 & 2 & 3 \\ 4 & 5 & 6 \\ 7 & 8 & 9 \end{bmatrix}`$
+   $A = \begin{bmatrix} 1 & 2 & 3 \\ 4 & 5 & 6 \\ 7 & 8 & 9 \end{bmatrix}$
```

Comment:
math error?

Reply:
I haven't touched anything about chapter 2, but this doesn't appear to be an error for me either